src/api/Feed.js

```diff
@@ -210,6 +234,23 @@ class Feed extends Base {
         });
     }
 
+    /**
+     * Fetches the tasks for a file
+     *
+     * @return {Promise} - the feed items
+     */
+    fetchTasksNew(): Promise<?Tasks> {
+        this.tasksNewAPI = new TasksNewAPI(this.options);
+
+        return new Promise(resolve => {
+            this.tasksNewAPI.getTasksForFile({
+                file: { id: this.id },
```

Comment:
It seems odd to spoof a file object here via an object literal. Can the `getTasksForFile` method accept a file id, instead?